### ISSUE TITLE
consider an option to add telemetry manually

### ISSUE THREAD

**Issue by eMBee** [an-anime-team/the-honkers-railway-launcher#100]
adding telemetry should not require me to enter a root password to run an unknown script. (the password prompt does not tell me what the installer will do as root)

consider an option to allow running this step manually by displaying the instructions how to do so:

"run this command as root or with sudo: /path/to/script"

concerned users can then inspect the script to verify that it is in fact benign.

this would also help as a workaround for those cases where the password prompt does not appear (#57)

**Comment by mkrsym1** [comment]
The "unknown script" is this: 
https://github.com/an-anime-team/the-honkers-railway-launcher/blob/71d217cc2658db99eeef5557858591f35f03f411/src/ui/main/disable_telemetry.rs#L10-L69 
Inspect it however you want. As you can see in the code, it's not actually on the disk but is instead generated dynamically and executed with `bash -c`. If you don't want to run it anyway, add the hostnames listed here to your /etc/hosts manually: 
https://github.com/an-anime-team/anime-game-core/blob/74a7c5a18cc4a870ff78e12d63ec136b3ef2d424/src/games/star_rail/consts.rs#L41-L52

**Comment by eMBee** [comment]
yes, i eventually realized that was this is possible. my suggestion is to make that visible to the user in the installer in some form. the current impression is that if i don't enter that password (or if the password prompt doesn't appear) then i am stuck without an alternative.
